### ISSUE TITLE
Terminal hangs after running tests and pressing ctrl+c

### ISSUE THREAD

**Issue by atif** [coryhouse/react-slingshot#544]
After npm start with all the environment loaded and app running in browser, whenever, I run tests by pressing enter and then try to terminate using Ctrl+C, it hangs the terminal i.e I am not able to see the cursor anymore. Anyone else facing the same issue?

**Comment by mirsujat** [comment]
It happened when npm cache limit exceeds. you may try 
npm cache clean --force               coupe of times then 
npm run start

thanks

**Comment by kwelch** [comment]
Are we sure this is not #120 creeping back up again?

Are you using a recent version of slingshot? 

Did cleaning cache work?

**Comment by atif** [comment]
Yes, I am using a recent version of slingshot. Issue persists after clearing cache as well. Here's the environment info:

npm: 5.7.1
node: 8.9.4

**Comment by coryhouse** [comment]
Hi Atif - Can you provide this info? 

Operating system:

Command line used:

**Comment by coryhouse** [comment]
And @kwelch - This is a different issue than #120 since that issue was resolved by us replacing npm-run-all with concurrently.

**Comment by atif** [comment]
@coryhouse  Operating System: MacOS Sierra 10.12.6, Command Line: MacOS Terminal 2.7.3 388.1.1)

**Comment by nemosupremo** [comment]
Hitting the same issue. I lose the terminal until I run the `reset` command. Here's a recording of what happens: https://asciinema.org/a/u8zuELPkIcmerVa0sKRw5vLPG

```
$ system_profiler SPSoftwareDataType

Software:

    System Software Overview:

      System Version: macOS 10.13.6 (17G65)
      Kernel Version: Darwin 17.7.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 4 days 3:26
```
